### DISCUSSION TITLE
performance test

### DISCUSSION THREAD

**Discussion by zhengjy** [naver/ngrinder#727] [Q&A] [answered]
versiom 3.5.3

input  >0 message
![image](https://user-images.githubusercontent.com/12868597/103271698-e273ff00-49f5-11eb-96c1-8a132341d342.png)

input ==0 message

![image](https://user-images.githubusercontent.com/12868597/103271758-10594380-49f6-11eb-9b24-a4179be641d6.png)

What's wrong with the operation  Or the problem ?

**Comment by donggyu04** [comment, marked as answer]
@zhengjy 

You must run agent first.
[This guide](https://github.com/naver/ngrinder/wiki/Installation-Guide#install-agents) will help you.
